src/test/sockman_tests.cpp

```diff
@@ -0,0 +1,152 @@
+// Copyright (c) 2021-2022 The Bitcoin Core developers
```

Comment:
535daaf15fd754335116d17833a45261cdff4e93

```suggestion
// Copyright (c) 2025-present The Bitcoin Core developers
```